nginx/default.conf

```
@@ -56,8 +56,8 @@ server {
     include fastcgi_params;
     fastcgi_intercept_errors on;
     fastcgi_pass php:9000;
-    fastcgi_buffers 16 16k;
-    fastcgi_buffer_size 32k;
+    fastcgi_buffers 128 4096k;
+    fastcgi_buffer_size 4096k;
```

To match production values from the operator:

```suggestion
    fastcgi_buffers 256 4k;
    fastcgi_buffer_size 48k;
```

---

Thanks @mjangda, just wondering if it was meant to be:

```
fastcgi_buffers 256 48k;
fastcgi_buffer_size 48k;
 ```

---

Nope, these were copied over directly from the config generator in production.

---

Great, again, thank you, will test this now!